packages/hooks/src/hooks.spec.tsx

```diff
@@ -95,33 +95,51 @@ describe('packages/hooks', () => {
   // Difficult to test a hook that measures changes to the DOM without having access to the DOM
   describe.skip('useMutationObserver', () => {}); //eslint-disable-line jest/no-disabled-tests
 
-  test('useViewportSize responds to updates in window size', async () => {
-    const { result, rerender } = renderHook(() => useViewportSize());
+  describe('useViewportSize', () => {
+    test('returns correct window dimensions on initial render', () => {
+      const mutableWindow: { -readonly [K in keyof Window]: Window[K] } =
+        window;
+      const initialHeight = 360;
+      const initialWidth = 480;
 
-    const mutableWindow: { -readonly [K in keyof Window]: Window[K] } = window;
-    const initialHeight = 360;
-    const initialWidth = 480;
+      mutableWindow.innerHeight = initialHeight;
+      mutableWindow.innerWidth = initialWidth;
 
-    mutableWindow.innerHeight = initialHeight;
-    mutableWindow.innerWidth = initialWidth;
+      const { result } = renderHook(() => useViewportSize());
 
-    window.dispatchEvent(new Event('resize'));
-    rerender();
-    await waitFor(() => {
       expect(result?.current?.height).toBe(initialHeight);
       expect(result?.current?.width).toBe(initialWidth);
     });
 
-    const updateHeight = 768;
-    const updateWidth = 1024;
+    test('responds to updates in window size', async () => {
+      const { result, rerender } = renderHook(() => useViewportSize());
 
-    mutableWindow.innerHeight = updateHeight;
-    mutableWindow.innerWidth = updateWidth;
+      const mutableWindow: { -readonly [K in keyof Window]: Window[K] } =
+        window;
+      const initialHeight = 360;
+      const initialWidth = 480;
 
-    window.dispatchEvent(new Event('resize'));
-    await waitFor(() => {
-      expect(result?.current?.height).toBe(updateHeight);
-      expect(result?.current?.width).toBe(updateWidth);
+      mutableWindow.innerHeight = initialHeight;
+      mutableWindow.innerWidth = initialWidth;
+
+      window.dispatchEvent(new Event('resize'));
+      rerender();
+      await waitFor(() => {
+        expect(result?.current?.height).toBe(initialHeight);
+        expect(result?.current?.width).toBe(initialWidth);
+      });
```

Comment:
probably could replace these lines with just the 2 expect statements from above

Reply:
Wonder if there's even value in asserting the initial response in this test now given that it's tested above. Perhaps it just makes sense to assert the one resize here?
```ts
    test('responds to updates in window size', async () => {
      const { result, rerender } = renderHook(() => useViewportSize());

      const mutableWindow: { -readonly [K in keyof Window]: Window[K] } =
        window;

      const updateHeight = 768;
      const updateWidth = 1024;

      mutableWindow.innerHeight = updateHeight;
      mutableWindow.innerWidth = updateWidth;

      window.dispatchEvent(new Event('resize'));
      await waitFor(() => {
        expect(result?.current?.height).toBe(updateHeight);
        expect(result?.current?.width).toBe(updateWidth);
      });
    });
```

Reply:
Pushed the change for this but happy to revert if you feel opposed to it.

Reply:
Since this was a nit and already approved, I'm going to merge. If more is needed here, happy to do so in a followup.